<?xml version="1.0"?>
<interface>
  <requires lib="gtk+" version="2.16"/>
  <!-- interface-naming-policy project-wide -->
  <object class="GtkWindow" id="gpx_viewer_window">
    <property name="title" translatable="yes">GPX Viewer</property>
    <property name="default_width">680</property>
    <property name="default_height">480</property>
    <property name="icon_name">gpxviewer</property>
    <accel-groups>
      <group name="accelgroup1"/>
    </accel-groups>
    <signal name="destroy_event" handler="on_destroy"/>
    <signal name="delete_event" handler="on_destroy"/>
    <child>
      <object class="GtkVBox" id="gpx_viewer_vbox">
        <property name="visible">True</property>
        <property name="orientation">vertical</property>
        <property name="spacing">6</property>
        <child>
          <object class="GtkMenuBar" id="menubar1">
            <property name="visible">True</property>
            <child>
              <object class="GtkMenuItem" id="menuitem1">
                <property name="visible">True</property>
                <property name="label" translatable="yes">_File</property>
                <property name="use_underline">True</property>
                <child type="submenu">
                  <object class="GtkMenu" id="menu1">
                    <property name="visible">True</property>
                    <child>
                      <object class="GtkImageMenuItem" id="menu_quit">
                        <property name="label">gtk-quit</property>
                        <property name="visible">True</property>
                        <property name="use_underline">True</property>
                        <property name="use_stock">True</property>
                        <property name="accel_group">accelgroup1</property>
                        <signal name="activate" handler="on_destroy"/>
                      </object>
                    </child>
                  </object>
                </child>
              </object>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkHBox" id="hbox1">
            <property name="visible">True</property>
            <property name="spacing">6</property>
            <child>
              <object class="GtkAlignment" id="alignment1">
                <property name="visible">True</property>
                <property name="yalign">0</property>
                <property name="yscale">0</property>
                <child>
                  <object class="GtkComboBox" id="routes_combo">
                    <property name="visible">True</property>
                    <property name="model">routes_store</property>
                    <signal name="changed" handler="routes_combo_changed_cb"/>
                    <child>
                      <object class="GtkCellRendererText" id="cellrenderertext1"/>
                      <attributes>
                        <attribute name="text">0</attribute>
                      </attributes>
                    </child>
                  </object>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkAlignment" id="alignment3">
                <property name="visible">True</property>
                <property name="xalign">0</property>
                <property name="yalign">0</property>
                <property name="yscale">0</property>
                <child>
                  <object class="GtkTable" id="table1">
                    <property name="visible">True</property>
                    <property name="n_rows">2</property>
                    <property name="n_columns">8</property>
                    <property name="column_spacing">6</property>
                    <property name="row_spacing">6</property>
                    <child>
                      <object class="GtkLabel" id="distance_title_label">
                        <property name="visible">True</property>
                        <property name="xalign">1</property>
                        <property name="label" translatable="yes">&lt;b&gt;Distance:&lt;/b&gt;</property>
                        <property name="use_markup">True</property>
                      </object>
                      <packing>
                        <property name="x_options">GTK_FILL</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="duration_title_label">
                        <property name="visible">True</property>
                        <property name="xalign">1</property>
                        <property name="label" translatable="yes">&lt;b&gt;Duration:&lt;/b&gt;</property>
                        <property name="use_markup">True</property>
                      </object>
                      <packing>
                        <property name="top_attach">1</property>
                        <property name="bottom_attach">2</property>
                        <property name="x_options">GTK_FILL</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="average_title_label">
                        <property name="visible">True</property>
                        <property name="xalign">1</property>
                        <property name="label" translatable="yes">&lt;b&gt;Average:&lt;/b&gt;</property>
                        <property name="use_markup">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">2</property>
                        <property name="right_attach">3</property>
                        <property name="x_options">GTK_FILL</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="moving_average_title_label">
                        <property name="visible">True</property>
                        <property name="xalign">1</property>
                        <property name="label" translatable="yes">&lt;b&gt;Moving average:&lt;/b&gt;</property>
                        <property name="use_markup">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">2</property>
                        <property name="right_attach">3</property>
                        <property name="top_attach">1</property>
                        <property name="bottom_attach">2</property>
                        <property name="x_options">GTK_FILL</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="distance_label">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">n/a</property>
                        <property name="ellipsize">end</property>
                      </object>
                      <packing>
                        <property name="left_attach">1</property>
                        <property name="right_attach">2</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="duration_label">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">n/a</property>
                        <property name="ellipsize">end</property>
                      </object>
                      <packing>
                        <property name="left_attach">1</property>
                        <property name="right_attach">2</property>
                        <property name="top_attach">1</property>
                        <property name="bottom_attach">2</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="average_label">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">n/a</property>
                        <property name="ellipsize">end</property>
                      </object>
                      <packing>
                        <property name="left_attach">3</property>
                        <property name="right_attach">4</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="moving_average_label">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">n/a</property>
                        <property name="ellipsize">end</property>
                      </object>
                      <packing>
                        <property name="left_attach">3</property>
                        <property name="right_attach">4</property>
                        <property name="top_attach">1</property>
                        <property name="bottom_attach">2</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="moving_averate_time_title_label">
                        <property name="visible">True</property>
                        <property name="xalign">1</property>
                        <property name="label" translatable="yes">&lt;b&gt;Moving time:&lt;/b&gt;</property>
                        <property name="use_markup">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">4</property>
                        <property name="right_attach">5</property>
                        <property name="top_attach">1</property>
                        <property name="bottom_attach">2</property>
                        <property name="x_options">GTK_FILL</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="moving_average_time_label">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">n/a</property>
                        <property name="ellipsize">end</property>
                      </object>
                      <packing>
                        <property name="left_attach">5</property>
                        <property name="right_attach">6</property>
                        <property name="top_attach">1</property>
                        <property name="bottom_attach">2</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="max_speed_title_label">
                        <property name="visible">True</property>
                        <property name="xalign">1</property>
                        <property name="label" translatable="yes">&lt;b&gt;Max. speed:&lt;/b&gt;</property>
                        <property name="use_markup">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">4</property>
                        <property name="right_attach">5</property>
                        <property name="x_options">GTK_FILL</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="max_speed_label">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="xpad">1</property>
                        <property name="label" translatable="yes">n/a</property>
                        <property name="ellipsize">end</property>
                      </object>
                      <packing>
                        <property name="left_attach">5</property>
                        <property name="right_attach">6</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="num_points_title_label">
                        <property name="visible">True</property>
                        <property name="xalign">1</property>
                        <property name="label" translatable="yes">&lt;b&gt;Gps points:&lt;/b&gt;</property>
                        <property name="use_markup">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">6</property>
                        <property name="right_attach">7</property>
                        <property name="x_options">GTK_FILL</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="num_points_label">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">n/a</property>
                        <property name="ellipsize">end</property>
                      </object>
                      <packing>
                        <property name="left_attach">7</property>
                        <property name="right_attach">8</property>
                      </packing>
                    </child>
                    <child>
                      <placeholder/>
                    </child>
                    <child>
                      <placeholder/>
                    </child>
                  </object>
                </child>
              </object>
              <packing>
                <property name="position">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkTable" id="table2">
                <property name="visible">True</property>
                <property name="n_rows">2</property>
                <property name="n_columns">2</property>
                <child>
                  <object class="GtkCheckButton" id="route_visible_check_button">
                    <property name="label" translatable="yes">Show</property>
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="receives_default">False</property>
                    <property name="draw_indicator">True</property>
                    <signal name="toggled" handler="route_set_visible"/>
                  </object>
                  <packing>
                    <property name="top_attach">1</property>
                    <property name="bottom_attach">2</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkAlignment" id="alignment2">
                    <property name="visible">True</property>
                    <property name="xalign">1</property>
                    <property name="xscale">0</property>
                    <child>
                      <object class="GtkEventBox" id="eventbox3">
                        <property name="visible">True</property>
                        <property name="visible_window">False</property>
                        <signal name="button_press_event" handler="playback_play_clicked"/>
                        <child>
                          <object class="GtkImage" id="play_button">
                            <property name="visible">True</property>
                            <property name="stock">gtk-media-play</property>
                            <property name="icon-size">1</property>
                          </object>
                        </child>
                      </object>
                    </child>
                  </object>
                </child>
                <child>
                  <object class="GtkEventBox" id="eventbox1">
                    <property name="visible">True</property>
                    <property name="visible_window">False</property>
                    <signal name="button_press_event" handler="playback_pause_clicked"/>
                    <child>
                      <object class="GtkImage" id="pause_clicked">
                        <property name="visible">True</property>
                        <property name="stock">gtk-media-pause</property>
                        <property name="icon-size">1</property>
                      </object>
                    </child>
                  </object>
                  <packing>
                    <property name="left_attach">1</property>
                    <property name="right_attach">2</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkEventBox" id="eventbox2">
                    <property name="visible">True</property>
                    <property name="visible_window">False</property>
                    <signal name="button_press_event" handler="playback_stop_clicked"/>
                    <child>
                      <object class="GtkImage" id="image2">
                        <property name="visible">True</property>
                        <property name="stock">gtk-media-stop</property>
                        <property name="icon-size">1</property>
                      </object>
                    </child>
                  </object>
                  <packing>
                    <property name="left_attach">1</property>
                    <property name="right_attach">2</property>
                    <property name="top_attach">1</property>
                    <property name="bottom_attach">2</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">2</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="position">1</property>
          </packing>
        </child>
        <child>
          <object class="GtkVPaned" id="main_view_pane">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="orientation">vertical</property>
            <child>
              <placeholder/>
            </child>
            <child>
              <placeholder/>
            </child>
          </object>
          <packing>
            <property name="position">2</property>
          </packing>
        </child>
        <child>
          <object class="GtkHBox" id="hbox2">
            <property name="visible">True</property>
            <child>
              <object class="GtkSpinButton" id="map_zoom_level">
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="tooltip_text" translatable="yes">Change the zoom level of the map.</property>
                <property name="max_length">2</property>
                <property name="width_chars">5</property>
                <property name="primary_icon_stock">gtk-zoom-in</property>
                <property name="adjustment">map_zoom_level_adjustment</property>
                <property name="climb_rate">1</property>
                <signal name="value_changed" handler="map_zoom_level_change_value_cb"/>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="position">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkSpinButton" id="smooth_factor">
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="tooltip_text" translatable="yes">Change the smoothing factor used when plotting the speed graph.</property>
                <property name="invisible_char">&#x25CF;</property>
                <property name="adjustment">smooth_factor_adjustment</property>
                <property name="climb_rate">1</property>
                <property name="numeric">True</property>
                <signal name="value_changed" handler="smooth_factor_change_value_cb"/>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="position">2</property>
              </packing>
            </child>
            <child>
              <object class="GtkCheckButton" id="show_marker_layer">
                <property name="label" translatable="yes">Show Waypoints</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">False</property>
                <property name="active">True</property>
                <property name="draw_indicator">True</property>
                <signal name="toggled" handler="show_marker_layer_toggled_cb"/>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="position">3</property>
              </packing>
            </child>
            <child>
              <object class="GtkStatusbar" id="statusbar2">
                <property name="visible">True</property>
                <property name="spacing">2</property>
              </object>
              <packing>
                <property name="pack_type">end</property>
                <property name="position">0</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="pack_type">end</property>
            <property name="position">3</property>
          </packing>
        </child>
      </object>
    </child>
  </object>
  <object class="GtkAccelGroup" id="accelgroup1"/>
  <object class="GtkListStore" id="routes_store">
    <columns>
      <!-- column-name Name -->
      <column type="gchararray"/>
      <!-- column-name Route -->
      <column type="gpointer"/>
    </columns>
  </object>
  <object class="GtkSizeGroup" id="heading_label_sizegroup">
    <widgets>
      <widget name="num_points_label"/>
      <widget name="max_speed_label"/>
      <widget name="moving_average_time_label"/>
      <widget name="moving_average_label"/>
      <widget name="average_label"/>
      <widget name="duration_label"/>
      <widget name="distance_label"/>
    </widgets>
  </object>
  <object class="GtkAdjustment" id="map_zoom_level_adjustment">
    <property name="value">5</property>
    <property name="lower">1</property>
    <property name="upper">19</property>
    <property name="step_increment">1</property>
    <property name="page_increment">1</property>
    <property name="page_size">1</property>
  </object>
  <object class="GtkAdjustment" id="smooth_factor_adjustment">
    <property name="value">4</property>
    <property name="lower">1</property>
    <property name="upper">100</property>
    <property name="step_increment">1</property>
    <property name="page_increment">1</property>
    <property name="page_size">1</property>
  </object>
</interface>
